scipy/special/_add_newdocs.py

```diff
@@ -8849,32 +9761,49 @@ def add_newdoc(name, doc):
     and n>=m, spheroidal parameter `c` and ``|x| < 1.0``. Requires
     pre-computed characteristic value.
 
+    Parameters
+    ----------
+    m : array_like
+        Nonnegative mode parameter m
+    n : array_like
+        Mode parameter n (>= m)
+    c : array_like
+        Spheroidal parameter
+    cv : array_like
+        Characteristic value
+    x : array_like
+        Real parameter (|x| < 1.0)
```

Comment:
```suggestion
        Real parameter (``|x| < 1.0``)
```

Reply:
not sure why these suggestions were showing up as "outdated" so that I could not add them here. I pushed a new commit with them instead